src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -41,12 +44,16 @@ public void parse_allCorrectInputs_success() throws ParseException, IOException
         Date expectedUntilDate = TestUtil.getDate(2017, Calendar.NOVEMBER, 30);
         Assert.assertEquals(expectedSinceDate, cliArguments.getSinceDate().get());
         Assert.assertEquals(expectedUntilDate, cliArguments.getUntilDate().get());
+
+        List<String> expectedFormats = Stream.concat(
+                ArgsParser.DEFAULT_FORMATS.stream(), Stream.of("html", "css", "js")).collect(Collectors.toList());
```

Comment:
The default formats won't be added in if we provide some other format and without them right? So don't do this concatenation and instead just provide them all straight.
Also, `Collections.list(...)` as shown above earlier.